tools/fleet-docker/Dockerfile

```diff
@@ -10,6 +10,5 @@ RUN addgroup -S fleet && adduser -S fleet -G fleet
 USER fleet
 
 COPY fleet /usr/bin/
-COPY fleetctl /usr/bin/
```

Comment:
Let me know if there's a reason to keep the fleetctl executable on the `fleetdm/fleet` docker image.